{{cookiecutter.project_slug}}/server/{{cookiecutter.project_slug}}/core/views.py

```diff
@@ -22,22 +22,19 @@
 {% else %}
 from .serializers import UserLoginSerializer, UserSerializer
 {% endif %}
-{% if cookiecutter.use_graphql == 'y' %}
+
 # Serve React frontend
 @ensure_csrf_cookie
 @never_cache
-def index(request):
-    return TemplateResponse(request, "index.html")
-{% elif cookiecutter.client_app.lower() == 'None' %}
-def index(request):
-    return redirect(to="/docs/swagger/")
-{% else %}
 def index(request):
     try:
-        return render(request, "index.html")
-    except TemplateDoesNotExist:
-        return render(request, "core/index-placeholder.html")
-{% endif %}
+        index_path = os.path.join(settings.BASE_DIR.replace('server', 'client'), 'dist/index.html')
+        if not os.path.exists(index_path):
+            raise FileNotFoundError
+        return TemplateResponse(request, "index.html")
+    except FileNotFoundError:
+        return TemplateResponse(request, "core/index-placeholder.html")
+
```

Comment:
I don't think you need the try/except here.
I think instead you can just return the placeholder template inside of `if not os.path.exists(index_path):`